x-pack/plugins/secops/common/graphql/introspection.json

```diff
@@ -227,6 +235,29 @@
         "enumValues": null,
         "possibleTypes": null
       },
+      {
+        "kind": "OBJECT",
+        "name": "SayMyName",
+        "description": "",
```

Comment:
I'd add the description even if this is throw away?

Does make me wonder if we have a `linter` to fail on this or other schema must-have's we add :-)

I'll add a discussion item to the backlog about this pre-commit hook even though I'm skiddish about pre-commit hooks:
https://github.com/cjoudrey/graphql-schema-linter